src/core/custom-config.cfg

```diff
@@ -0,0 +1,4 @@
+//Update default settings in Firefox's about:config
+//TODO: Uncomment the following line to use the uproxy landing page instead of uproxy.org when ready
+//pref("startup.homepage_welcome_url", "file:///C:/Program Files/Mozilla Firefox/uproxy.html");
+pref("startup.homepage_welcome_url", "https://www.uproxy.org");
```

Comment:
I'm a bit worried about this.  It'd immediately identify uProxy users in some countries. I saved a copy of uproxy.org to a uproxy.html (plus sub-dir for stuff) here.
[uproxy-savedir.zip](https://github.com/uProxy/uproxy-windows-installer/files/226720/uproxy-savedir.zip).  So that could work as v1 of the landing page.